test/__snapshots__/katex-spec.js.snap

```diff
@@ -0,0 +1,44 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`An implicit group parser should work within optional groups: \\sqrt[\\small 3]{x} 1`] = `
+Array [
+  ParseNode {
+    "mode": "math",
+    "type": "sqrt",
+    "value": Object {
+      "body": ParseNode {
+        "mode": "math",
+        "type": "ordgroup",
+        "value": Array [
+          ParseNode {
+            "mode": "math",
+            "type": "mathord",
+            "value": "x",
+          },
+        ],
+      },
+      "index": ParseNode {
+        "mode": "math",
+        "type": "ordgroup",
+        "value": Array [
+          ParseNode {
+            "mode": "math",
+            "type": "sizing",
+            "value": Object {
+              "size": 5,
+              "value": Array [
+                ParseNode {
+                  "mode": "math",
+                  "type": "textord",
+                  "value": "3",
+                },
+              ],
+            },
+          },
+        ],
+      },
+      "type": "sqrt",
```

Comment:
(optional) I don't know whether an arbitrary strict order is enforced here, but if not, moving this up would make it much easier to see which level it is in.

Reply:
I can set up a custom formatter for the snapshots to always put `"type"` first.